doc/manifest/schema/1.0.0/installer.md

```diff
@@ -0,0 +1,444 @@
+
+</details>
+
+### InstallerType
+<details>
+ <summary>Enumeration of supported installer types. InstallerType is required in either root level or individual Installer level</summary>
+
+ #### Required Field
+
+ This key represents the installer type for the package. The Windows Package Manager supports MSIX, MSI, and executable installers. Some well known formats provide standard sets of installer switches to provide different installer experiences.
+
+ >Note: The Windows Package Manager defaults to the install mode providing install progress. A best practice is to determine if one of the supported installer technologies was used to build an installer with the .exe file extension. The [Windows Package Manager Manifest Creator](https://github.com/microsoft/winget-create) tool can be used to determine if one of the known tools was used to build an installer with the .exe file extension.
+
+ >Note: The Windows Package Manager does not support loose executables with the .exe or .com file extension directly. Compressed files containing installers, or loose executables are also not supported.
+ 
+ >Note: This key may be present in the root of the manifest as the default value for all installer nodes. This key may also be present in an individual installer node as well. If this key is in the manifest root and in an installer node, the value in the installer node will apply.
+
+</details>
+
+### Scope
+<details>
+ <summary>Scope indicates if the installer is per user or per machine</summary>
+
+ #### Optional Field
+
+ This key represents the scope the package is installed under. The two configurations are "user" and "machine". Some installers support only one of these scopes while others support both via arguments passed to the installer using "InstallerSwitches".
+ 
+ >Note: This key may be present in the root of the manifest as the default value for all installer nodes. This key may also be present in an individual installer node as well. If this key is in the manifest root and in an installer node, the value in the installer node will apply.
+
+</details>
+
+### InstallModes
+<details>
+ <summary>List of supported installer modes</summary>
+
+ #### Optional Field
+
+ This key represents the install modes supported by the installer. The Microsoft community package repository requires a package support "silent" and "silent with progress". The Windows Package Manager also supports "interactive" installers.
+
+ >Note: Some installers will attempt to install missing dependencies. If these dependencies require user interaction, the package will not be allowed into the Microsoft community package repository.
+
+ >Note: This key may be present in the root of the manifest as the default value for all installer nodes. This key may also be present in an individual installer node as well. If this key is in the manifest root and in an installer node, the value in the installer node will apply.
```

Comment:
Note: not used by winget client yet

Reply:
what happens if I try to specify one of the modes not supported by the installer?

Reply:
Regarding switches selection, we have fallback logic to pick the most appropriate one. We don't force the installation to quit if desired behavior cannot be met, for now.